### ISSUE TITLE
Interpolation Information

### ISSUE THREAD

**Issue by KathiSchleidt** [FAIRiCUBE/data-requests#58]
In the current update of the Inventory.xls, we can provide interpolation by Dimension

When I compare this with the options for providing interpolation information via Coverage encoding, I can only see this under rangeType: https://schemas.opengis.net/cis/1.1/gml/examples-1.1/70_2D_interpolation.xml

**Comment by pebau** [comment]
yes, you caught me - the insight that different axes may have different interpolations did not make it into CIS 1.1, that insight is younger.

So I wanted to collect unbiased information and then see - with a small hope was that we get the same on x/y as on t, but the game is still ongoing. Should they differ then we need to put our pretty little minds together and find a way forward (possibly an input to the upcoming 19123-2 = CIS work).

**Comment by KathiSchleidt** [comment]
Could you please provide examples where the interpolation is different dependent on the dimension? Would help me better understand what you're trying to get to.

If you state that the lack of interpolation information per dimension is a deficit of CIS encoding, I'm sure you also have the examples to underpin this!

**Comment by KathiSchleidt** [comment]
Additional insight from discussions today with @sonjastndl - think we actually have to differentiate between interpolation and aggregation!

Example: gridded dataset with average temperature
- Spatially: 
  - I can (probably) interpolate between locations (though does depend on the underlying topology)
  - I can aggregate to a lower spatial resolution by taking the average of these averages
- Temporally:
  - I cannot interpolate the daily averages to finer values, e.g. value at noon vs. value at midnight
  - I can aggregate the daily average to a weekly average

Based on this insight, in addition to providing interpolation information explicitly by dimension, don't we also need to provide aggregation information?

**Comment by KathiSchleidt** [comment]
@pebau any feedback on this insight? In lieu thereof, I've added this to the Inventory sheet
